distros/ronin/apps.list

```diff
@@ -0,0 +1,29 @@
+apps/bluetooth
+apps/bookmark
+apps/calendar
+apps/camera
+apps/clock
+apps/default_theme
+apps/download
+apps/email
+apps/findmydevice
+apps/fl
+apps/ftu
+apps/gallery
+apps/homescreen
+apps/keyboard
+apps/marketplace.firefox.com
+apps/music
+apps/network-alerts
+apps/operatorvariant
+apps/pdfjs
+apps/ringtones
```

Comment:
I'm going to test. I don't know if it is used only for ringtones, or also for other sounds.